src/validations/access/common/profile.ts

```diff
@@ -0,0 +1,110 @@
+import { Avatar, Entity } from '@dcl/schemas'
+import { parseUrn } from '@dcl/urn-resolver'
+import {
+  ContentValidatorComponents,
+  DeploymentToValidate,
+  ItemsOwnership,
+  NamesOwnership,
+  OK,
+  ValidateFn,
+  ValidationResponse,
+  validationFailed
+} from '../../../types'
+import { isOldEmote } from '../../profile'
+import { ADR_74_TIMESTAMP, ADR_75_TIMESTAMP } from '../../timestamps'
+import { validateAfterADR75 } from '../../validations'
+
+export function createNamesOwnershipValidateFn(
+  { externalCalls }: Pick<ContentValidatorComponents, 'externalCalls'>,
+  namesOwnership: NamesOwnership
+): ValidateFn {
+  async function validateFn(deployment: DeploymentToValidate) {
+    const ethAddress = externalCalls.ownerAddress(deployment.auditInfo)
+    const names = deployment.entity.metadata.avatars
```

Comment:
I think names have to be lowercase when querying the-graph

Reply:
Sorry, no... not the case. The query needs to be done in the exact same case as the name was minted.

Reply:
On the contrary names in the graph are case sensitive :(. In addition, this is just moved code, no logic added